### ISSUE TITLE
usingDirectivesPlacement should be ignored unless fixing SA1200

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#2027]
Currently the using directives code fix uses the current `usingDirectivesPlacement` setting when organizing using directives for any supported diagnostic. This behavior will not correctly handle cases where diagnostic suppressions are used to disable particular instances of SA1200.

To correct this issue, the code fix should operate as though `usingDirectivesPlacement` were set to `ignore` when fixing any diagnostic other than SA1200.
### Example code

``` csharp
using System;
using System.Reflection;
using System.Net;

namespace MyNamespace
{
    using System.ServiceModel.Channels;
}
```
### Operation

Fix all violations of SA1210.
### Expected result

``` csharp
using System;
using System.Net;
using System.Reflection;

namespace MyNamespace
{
    using System.ServiceModel.Channels;
}
```
### Actual result

``` csharp
using System;
using System.Net;
using System.Reflection;
using System.ServiceModel.Channels;

namespace MyNamespace
{
}
```

**Comment by vweijsters** [comment]
Grabbing this
